tests/test_processing.py

```diff
@@ -175,7 +175,7 @@ def test_adapt_freq_add_dims(use_dask, random):
         prref = xr.where(pr < 10, pr / 20, pr)
     sim_ad, pth, _dP0 = adapt_freq(prref, prsim, thresh="1 mm d-1", group=group)
     assert set(sim_ad.dims) == set(prsim.dims)
-    assert "lat" not in pth.dims
+    assert "lat" in pth.dims
```

Comment:
now the adapt-freq works a bit differently with add-dims than the rest of training. ADD-DIMS is considered like something that should not be pooled for a common adapt freq. 

I think it makes sense, since adapt-freq is really a kind of pre-processing. Before, it was not even included in the adjustment procedures. This was changed because inclusion in the grouping structure was needed.